DSLink.NET/WebSocketSharpConnector.cs

```diff
@@ -37,19 +32,19 @@ public WebSocketSharpConnector(AbstractContainer link) : base(link)
             {
                 if (e.WasClean)
                 {
-                    _link.Logger.Info(string.Format("WebSocket was closed cleanly with code {0}, and reason \"{1}\"", e.Code, e.Reason));
+                    Logger.Info(string.Format("WebSocket was closed cleanly with code {0}, and reason \"{1}\"", e.Code, e.Reason));
```

Comment:
I'm wondering, what's the point of injecting the logger if you're just using statically?